chispa/dataframe_comparer.py

```diff
@@ -21,15 +21,36 @@ def assert_df_equality(df1, df2, ignore_nullable = False, transforms=[]):
         assert_schema_equality(s1, s2)
     rows1 = df1.collect()
     rows2 = df2.collect()
-    if rows1 != rows2:
-        t = PrettyTable(["df1", "df2"])
+    if allow_nan_equality:
```

Comment:
Is there a way you could restructure this so that `if allow_nan_equality` is checked on specific row comparisons? It would cut down on some code duplication.

Reply:
Created a separate issue, agree this is important